### PR TITLE
Update UI->One

### PR DESCRIPTION
**Pull request by iceXshadow**
Hey, thanks for assigning me this task. I made some changes here, let me know if you like it or not and if you have any recommendations in mind

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -4,24 +4,70 @@
...
...uX/n143bUrmkKgf6r+ThFPEU8Rz1sjnj3fSIe+QV6EhFL5ZCAIO7OpXJjo1aJgjXA8mIjdA+ZuyDsWS3HNYia87b8HQqZ05j0x6BceWBwnM/X36lkUb1hazIxCZKrubFdBEAThuQnnxO0QxPVe6HeF3V7VFkjyVPM/RTxFPEU874WENg8JxZz5MxtEHsLm/kt34iQsiWDdA8L9xhzWOoqrUMRt0nmi/HdhUcgKpdxDes++RZEFEM+OlowgCIJQgYVFIcBbr6JBVnUmwiniKeIp4nlv5LN1bfUzz2L1nRSoTFKoCygEh7Xzbn9LurF+JhZ7TJFVLej3K4vCRxiHU4e05xbLegVBEAThu1AQ4RTjEfEU8RTxfAgi2rCozrZRaLskMfWe01+2O8AeBCOUtcz85z90d4QzGe5jgh1TX4uvunSCIAjCNwBCRgsRThFPEU8Rz0cmoIEQYnEElba2b3zFmT+rSfMeuxYlyVf+c8yPHPjvhetj7ISt/2TnHQIfp4hhLX3NqsRXEARBOAYri8H/hWZyiniKeIp4Pg35DNG1P19fnPCNnYgunQROLAoUrc/4uW2LSqZrJzxo9tdN/x4cmyl8ZoTnY+TrdaDLIQiCINS0s0GMcaZLIeIp4ini+ewktEljUQIp5NLDsEmOz10CAuLr32OsBhr/1yKhgiAIgiDcM3exv1oCocVIpbQiniKeIp5CGSG0vyWXgQCG7E4ohQ3EkFVHg2jRHKTx2FIRn+c4cEKblxFSJ8A4jqFFwSJBEARBEIRbI5nBR9qW0v7xZZa6LCKeIp4insJphDRkIzFXcSMQFDbVPz//8efrxwnkdqeE98/PAhEt0jIUL81tW5wbillbYUNHj57mhQqCIAiC8B2EU/M2RTwFEU/hAuQz9IOGDCREgOYWewMx4gJlJS0nl/kxhPRQKYor5fbt+YRwBEEQBEH4PoIJx3QrDhR8HpXQingKIp7CZYhnIJOvvuGak8sqZU/MVVw4AZ0k5HFdZ7P2jGfIhk6ZxLoabiDAGRFdzogKgiAIgiAcSzDn7sNsfA/3YzAWbiWiKeIpiHgK30NEMQKlDtkLxDP0PhROFgPxnILQVvWJOkkNmdaZE943NwyjdPN3khoWTs+/Np2MTo44TkEQBEEQnguf5mnWqcISRDyFXfy/AAMApABRCDNfnWYAAAAASUVORK5CYII=);
```
Comment (Sanjeev-Kumar78) [inline styles/style.css:40]: I think it's better to use a .png file instead of base64 and save the background image in the asset folder. @iceXshadow

**Review by Sanjeev-Kumar78 — CHANGES_REQUESTED** [review]

File: styles/style.css (hunk 1 above)
Comment (iceXshadow) [reply styles/style.css:40]: Okay, I'll change it to png

**Comment by iceXshadow** [comment]
Hey I did the requested changes and some to table, kindly review and comment

**Comment by Sanjeev-Kumar78** [comment]
In this af6612e7fd9c09308cb19a53e936db76efed5b5f commit request your transparent PNG background is not displaying above the background color.
Here's a look of previous commit:
image
Take a look at the border.
***
image
In this, the below chalk is displaying try to use internal CSS for chalk in home.html and align all elements in the center.

**Comment by iceXshadow** [comment]
Sure, I'll fix it

**Review by Sanjeev-Kumar78 — APPROVED** [review]

**Comment by iceXshadow** [comment]
okay

**Comment by iceXshadow** [comment]
> In this [af6612e](https://github.com/Sanjeev-Kumar78/TeachersDay-Invitation/commit/af6612e7fd9c09308cb19a53e936db76efed5b5f) commit request your transparent PNG background is not displaying above the background color. Here's a look of previous commit: image Take a look at the border.
> can you be more specific about this?

**Comment by Sanjeev-Kumar78** [comment]
image
Take a look at background. It's smaller than the previous image.

**Comment by iceXshadow** [comment]
> image
> 
> * Nothing Border is good. Just fix this.
>   Take a look at background. It's smaller than the previous image.

>Fixed

**Pull request merged**